src/libs/BankAccountUtils.ts

```diff
@@ -1,10 +1,9 @@
+import {Str} from 'expensify-common';
 import type {OnyxEntry} from 'react-native-onyx';
 import type * as OnyxTypes from '@src/types/onyx';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
 function getDefaultCompanyWebsite(session: OnyxEntry<OnyxTypes.Session>, user: OnyxEntry<OnyxTypes.User>): string {
-    // temporarily always return https:// to fix https://github.com/Expensify/App/issues/47227 until https://github.com/Expensify/App/issues/45278 is resolved.
-    return 'https://';
+    return user?.isFromPublicDomain ? 'https://' : `https://www.${Str.extractEmailDomain(session?.email ?? '')}`;
```

Comment:
I don't have a private domain email, so I manually change the condition here to return the email domain for testing.